packages/bits/src/lib/table/table-cell/table-cell.directive.ts

```diff
@@ -3,6 +3,7 @@ import { ChangeDetectorRef, Directive, ElementRef, HostBinding, HostListener, In
 import _get from "lodash/get";
```

Comment:
I'm wondering if the changes in this file are necessary. Maybe they are necessary, but I would think we could just set the header cells and the body cells would follow suit automatically. Do you mind checking?

Reply:
If we dont keep this then the header and body column do not stay in sync.

Reply:
okay, thanks